docs/pages/aggregate.md

```diff
@@ -653,7 +653,7 @@ Or for test purposes the `FrozenClock`, which always returns the same time.
 
 In some cases, it makes sense to split an aggregate into several smaller aggregates.
 This can be the case if the aggregate becomes too large or if the aggregate is used in different contexts.
-We currently support two patterns for this: Micro Aggregates and Child Aggregates (experimental).
+We currently support only one pattern for this: Micro Aggregates.
```

Comment:
I would rephrase this, as it contextually now not much sense to talk about only one pattern.

```suggestion
For these cases you can use Micro Aggregates.
```